### ISSUE TITLE
nullable type is not probably handled.

### ISSUE THREAD

**Issue by aBaiLinrun** [kminami/apib2swagger#49]
**Problem** :  When `nullable` type is specified along with primitive type or `required` `optional` type, it was not probably handled, or we say missing handling. 

**Solution** : Add `nullable` type handling in `mson` schema convert process.

**apib file**
```markdown
# Group Users

## A user [/user]

### Get one user [GET]
+ Response 200 (application/json)
  + Attributes (User)
+ Response 500 (application/json)

## Some users [/users]

### Get a collection of users [GET]
+ Response 200 (application/json)
  + Attributes (array[User])
+ Response 500 (application/json)

# Data Structures

## User (object)
+ id (string, required)
+ name (string, nullable, required)

## Users (object)
+ `users`(array[User], nullable, required)
```

**json file**
```json
{
    "swagger": "2.0",
    "info": {
        "title": "",
        "version": "",
        "description": ""
    },
    "paths": {
        "/user": {
            "get": {
                "responses": {
                    "200": {
                        "description": "OK",
                        "headers": {},
                        "examples": {
                            "application/json": {
                                "id": "",
                                "name": null
                            }
                        },
                        "schema": {
                            "type": "object",
                            "properties": {
                                "id": {
                                    "type": "string"
                                },
                                "name": {
                                    "type": [
                                        "string",
                                        "null"
                                    ]
                                }
                            },
                            "required": [
                                "id",
                                "name"
                            ]
                        }
                    },
                    "500": {
                        "description": "Internal Server Error",
                        "headers": {},
                        "examples": {}
                    }
                },
                "summary": "Get one user",
                "operationId": "Get one user",
                "description": "",
                "tags": [
                    "Users"
                ],
                "parameters": [],
                "produces": [
                    "application/json"
                ]
            }
        },
        "/users": {
            "get": {
                "responses": {
                    "200": {
                        "description": "OK",
                        "headers": {},
                        "examples": {
                            "application/json": [
                                {
                                    "id": "",
                                    "name": null
                                }
                            ]
                        },
                        "schema": {
                            "type": "array",
                            "items": {}
                        }
                    },
                    "500": {
                        "description": "Internal Server Error",
                        "headers": {},
                        "examples": {}
                    }
                },
                "summary": "Get a collection of users",
                "operationId": "Get a collection of users",
                "description": "",
                "tags": [
                    "Users"
                ],
                "parameters": [],
                "produces": [
                    "application/json"
                ]
            }
        }
    },
    "definitions": {
        "A user": {},
        "Some users": {},
        "User": {
            "type": "object",
            "required": [
                "id",
                "name"
            ],
            "properties": {
                "id": {
                    "type": "string"
                },
                "name": {
                    "type": "string" // <= it should be ["string", "null"], but the null part is missing.
                }
            }
        },
        "Users": {
            "type": "object",
            "required": [
                "users"
            ],
            "properties": {
                "users": {
                    "type": "array", // <= it should be ["array", "null"], but the null part is missing.
                    "items": {}
                }
            }
        }
    },
    "securityDefinitions": {},
    "tags": [
        {
            "name": "Users"
        }
    ]
}
```

**Comment by kminami** [comment]
I have merged #48 and release it as v1.12.0.
It should resolve this issue. Thanks!

**Comment by aBaiLinrun** [comment]
Thank you!
